### PR TITLE
Delete federation canary pull job

### PR DESCRIPTION
**Pull request by fejta**
/assign @madhusudancs @krzyzacy 

Need to recover/delete k8s-jkns-pr-cnry-e2e-gce-fdrtn

Job has not passed in 231 days: http://velodrome.k8s.io/dashboard/db/bigquery-metrics?orgId=1

### CODE REVIEW

**Comment by krzyzacy** [comment]
I think folks deleted and added this job back fairly recently
/assign @shashidharatd

File: jobs/config.json [hunk 1]
```diff
@@ -8684,47 +8684,6 @@
       "sig-node"
     ]
   },
-  "ci-kubernetes-pull-gce-federation-deploy": {
```
Comment (shashidharatd) [inline jobs/config.json]: This job is required currently and is the daily cluster recycler for federation presubmit job.

File: jobs/env/ci-kubernetes-pull-gce-federation-deploy.env [hunk 2]
```diff
@@ -1,13 +0,0 @@
-# NOTE: Although this job is a dependency for a pull job, this is
```
Comment (shashidharatd) [inline jobs/env/ci-kubernetes-pull-gce-federation-deploy.env]: This env file also is required.

**Comment by shashidharatd** [comment]
@fejta, i am taking care of removing federation canary jobs as part of https://github.com/kubernetes/test-infra/pull/5187. Also i have taken care of the comments i mentioned above. So can we close this PR?

**Comment by fejta** [comment]
Thanks!

**Pull request closed without merging**